src/test/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtilsUnitTest.java

```diff
@@ -61,12 +61,12 @@ public void testCompareStringent(final String expectedFile, final String actualF
 
     @Test(dataProvider = "testCRAMContentsSucceed")
     public void testAssertCRAMContentsSucceed(File putativeCRAMFile) {
-        SamAssertionUtils.assertCRAMContents(putativeCRAMFile);
+        SamAssertionUtils.assertCRAMContents(putativeCRAMFile.toPath());
     }
 
     @Test(dataProvider = "testCRAMContentsFail", expectedExceptions=AssertionError.class)
     public void testAssertCRAMContentsFail(File putativeCRAMFile) {
-        SamAssertionUtils.assertCRAMContents(putativeCRAMFile);
+        SamAssertionUtils.assertCRAMContents(putativeCRAMFile.toPath());
```

Comment:
Test the file-based overloads as well

Reply:
There's no file-based overload of `assertCRAMContents` anymore.